.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: Scheduled Build
+
+on:
+  schedule:
+  - cron: '0 8 * * 6' # Scheduled the job to run every Saturday at 8 AM UTC
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Cache Nuget packages
+        uses: actions/cache@ab5e6d0c87105b4c9c2047343972218f562e4319 #v4.0.1
+        with:
+          path: ~/.nuget/packages
+          key: ${{ runner.os }}-nuget-${{ hashFiles('**/Directory.Packages.props') }}
+          restore-keys: |
+            ${{ runner.os }}-nuget-
+
+      - name: Build
+        id: build_step
+        run: |
+          ./build.ps1 Build -Configuration Debug || {
+            echo "error_message=Command failed: ./build.ps1 Build -Configuration Debug: Permission denied" >> $GITHUB_ENV
+            exit 1
+          }
+        shell: bash
+
+      - name: Run Unit Tests
+        id: unit_test_step
+        run: |
+         ./build.ps1 UnitTest -Configuration Debug || {
+            echo "error_message=Command failed: ./build.ps1 UnitTest -Configuration Debug: Permission denied" >> $GITHUB_ENV
+            exit 1
+          }
+        shell: bash
+
+      - name: Set error message output
+        if: failure()
+        id: set_error_message
+        run: |
+          echo "::set-output name=error::${{ env.error_message }}"
+
+      - name: Notify Slack on success
+        if: success()
```

Comment:
I thought we suggested no notification on success? I'm open either way but I thought we discussed that we only wanted slack notifications on failure.  Am I misremembering?

Reply:
Slack notification on success will probably become overwhelming.

Reply:
Oh, this is just the scheduled job. That's not such a big deal then - only one success message a week. I kind of like that.